Sources/swift-container-plugin/Empty.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftContainerPlugin open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftContainerPlugin project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftContainerPlugin project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+// This is an empty source file used to make SwiftContainerPluginDocumentation a valid
+// documentation build target.
+
+// SwiftContainerPluginDocumentation is an otherwise empty target that includes high-level,
+// user-facing documentation about using the Swift Container Plugin from the command-line.
```

Comment:
Joe suggested including the `swift plugin ... preview-documentation` command here.

Reply:
`swift package --disable-sandbox preview-documentation --target swift-container-plugin`

Reply:
Fixed in https://github.com/apple/swift-container-plugin/pull/99/commits/8a91eb0fccbbe46d0f959ad51bced50543261dd6 and https://github.com/apple/swift-container-plugin/pull/99/commits/af35361ad47e4f0c27672bc3172a5d07e9a31245